io.github.suchnsuch.Tangent.yaml

```diff
@@ -0,0 +1,55 @@
+id: io.github.suchnsuch.Tangent
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node22
+# Electron doesn't use a traditional locale format
+separate-locales: false
+writable-sdk: true
+command: run.sh
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --share=network
```

Comment:
the file dialogue doesn't seem to work with portals.

Reply:
![image](https://github.com/user-attachments/assets/69d2b2bc-55e8-4ed2-8fbc-e0f2d24294ca)

Reply:
That's strange, it works on my system.

![image](https://github.com/user-attachments/assets/a129eba5-f2bd-488d-bed5-1478ab6f78c0)

Reply:
hmm weird what version of the portal do you have?

Reply:
`xdg-desktop-portal`: `1.18.4`
`xdg-desktop-portal-gnome`: `47.1`
`xdg-desktop-portal-gtk`: `1.15.1`

Reply:
that's same as mine.

Reply:
Weird that yours shows the legacy GNOME file picker

Reply:
Yea probably something related to Electron, what version are you using?

Reply:
Could be https://github.com/electron/electron/issues/43819

Reply:
Yea looks like it, 

```
[3:1114/193834.107971:VERBOSE1:select_file_dialog_linux_portal.cc(300)] File chooser portal available: no
```

If you are using >=32.1, might be better to downgrade it until the bug is solved.

Reply:
Yup, this seems like that's it. We just switched to 33.0.2 in the latest version :smile:

Reply:
FYI, I was able to reproduce this on one of my systems. We have decided to wait until Electron 34 releases instead of downgrading to a previous version.

![image](https://github.com/user-attachments/assets/ef0388e5-8797-4f01-af8d-10fd7b008cd4)

Reply:
> We have decided to wait until Electron 34 releases

Ok ping when the version using 34 is out, this should be working before accepting the submission.